Lesson5/src/app/pages/Comment.js

```diff
@@ -0,0 +1,25 @@
+import React, {Component} from 'react';
+import axios from 'axios';
+import CommentProfile from '../components/Comment';
+
+export default class Comment extends Component {
+	constructor(props){
+		super(props);
+		this.state = {
+			comments: null
+		}
+	}
+	
+	render(){
+		return(
+		<div>
+			{this.state.comment && <CommentProfile {...this.state.comments} />}
```

Comment:
```suggestion
			{this.state.comments && <CommentProfile {...this.state.comments} />}
```
вот теперь коммент будет отдельно отображаться:) 
хотя лучше поменять свойства `this.state` на `comment` (вместо `comments`)